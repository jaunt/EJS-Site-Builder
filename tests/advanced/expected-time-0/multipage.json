["index","index","index","posts","posts","posts","posts","posts","posts","posts","posts","posts","preGenerate.js","random","random","random","random","random","widgets/label"]