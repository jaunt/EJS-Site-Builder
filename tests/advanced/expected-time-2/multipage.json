["index","index","index","index","index","index","postGenerate.js","postGenerate.js","postGenerate.js","postGenerate.js","posts","posts","posts","posts","posts","posts","posts","posts","posts","posts","posts","posts","posts","posts","posts","posts","posts","posts","preGenerate.js","preGenerate.js","random","random","random","random","random","random","random","random","random","random","widgets/label","widgets/label"]