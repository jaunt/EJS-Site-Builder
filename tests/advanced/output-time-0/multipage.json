[{"page":"/","path":"/home/steve/projects/airfry/tests/advanced/output/index.html"},{"page":"/","path":"/home/steve/projects/airfry/tests/advanced/output/main.js"},{"page":"/posts/first","path":"/home/steve/projects/airfry/tests/advanced/output/posts/first/index.html"},{"page":"/posts/first","path":"/home/steve/projects/airfry/tests/advanced/output/posts/first/index.js"},{"page":"/posts/fourth","path":"/home/steve/projects/airfry/tests/advanced/output/posts/fourth/index.html"},{"page":"/posts/fourth","path":"/home/steve/projects/airfry/tests/advanced/output/posts/fourth/index.js"},{"page":"/posts/second","path":"/home/steve/projects/airfry/tests/advanced/output/posts/second/index.html"},{"page":"/posts/second","path":"/home/steve/projects/airfry/tests/advanced/output/posts/second/index.js"},{"page":"/posts/third","path":"/home/steve/projects/airfry/tests/advanced/output/posts/third/index.html"},{"page":"/posts/third","path":"/home/steve/projects/airfry/tests/advanced/output/posts/third/index.js"},{"page":"/random/page0","path":"/home/steve/projects/airfry/tests/advanced/output/random/page0/index.html"},{"page":"/random/page1","path":"/home/steve/projects/airfry/tests/advanced/output/random/page1/index.html"},{"page":"/random/page2","path":"/home/steve/projects/airfry/tests/advanced/output/random/page2/index.html"},{"page":"index","path":"/home/steve/projects/airfry/tests/advanced/output/README.md"},{"page":"posts","path":"/home/steve/projects/airfry/tests/advanced/output/posts/data.json"},{"page":"posts","path":{"time":"2000/01/01"}},{"page":"preGenerate.js","path":"/home/steve/projects/airfry/tests/advanced/output/version.json"},{"page":"widgets/label","path":"/home/steve/projects/airfry/tests/advanced/output/js/widgets/label.js"}]