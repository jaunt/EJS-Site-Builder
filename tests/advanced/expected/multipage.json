{"output":{"html":{"/":"/home/steve/projects/airfryts/tests/advanced/output/index.html","/posts/first":"/home/steve/projects/airfryts/tests/advanced/output/posts/first/index.html","/posts/fourth":"/home/steve/projects/airfryts/tests/advanced/output/posts/fourth/index.html","/posts/second":"/home/steve/projects/airfryts/tests/advanced/output/posts/second/index.html","/posts/third":"/home/steve/projects/airfryts/tests/advanced/output/posts/third/index.html","/random/page1":"/home/steve/projects/airfryts/tests/advanced/output/random/page1/index.html","/random/page2":"/home/steve/projects/airfryts/tests/advanced/output/random/page2/index.html","/random/page0":"/home/steve/projects/airfryts/tests/advanced/output/random/page0/index.html"},"entry":{"/":"/home/steve/projects/airfryts/tests/advanced/output/main.js","/posts/first":"/home/steve/projects/airfryts/tests/advanced/output/posts/first/index.js","/posts/fourth":"/home/steve/projects/airfryts/tests/advanced/output/posts/fourth/index.js","/posts/second":"/home/steve/projects/airfryts/tests/advanced/output/posts/second/index.js","/posts/third":"/home/steve/projects/airfryts/tests/advanced/output/posts/third/index.js"},"lib":{"widgets/label":"/home/steve/projects/airfryts/tests/advanced/output/js/widgets/label.js"},"json":{"preGenerate.js":"/home/steve/projects/airfryts/tests/advanced/output/version.json","index":"/home/steve/projects/airfryts/tests/advanced/output/README.md","posts":"/home/steve/projects/airfryts/tests/advanced/output/posts/data.json","postGenerate.js":"/home/steve/projects/airfryts/tests/advanced/output/multipage.json"},"outData":{"posts":{"time":"2021-11-16T04:52:02.812Z"}}}}